src/Makefile.qt.include

```diff
@@ -868,6 +869,8 @@ pivx_qt_clean: FORCE
 
 pivx_qt : qt/pivx-qt$(EXEEXT)
 
+$(QT_PIVX_CPP) : $(QT_PIVX_MOC)
+
```

Comment:
Unneeded, these are not defined.

Reply:
ahh yeah, that's leftover from previous work, removing